instrumentation/go.mongodb.org/mongo-driver/mongo/otelmongo/test/mongo_test.go

```diff
@@ -101,6 +122,11 @@ func TestDBCrudOperation(t *testing.T) {
 		} else {
 			title = title + "/includeCommand"
 		}
+		if tc.sanitiser != nil {
+			title = title + "/customSanitiser"
+		} else {
+			title = title + "/defaultSanitiser"
+		}
```

Comment:
this is very unreadable. Can we change the title is setup in `tt` declaration? Same comment applies for the logic which appends `/excludeCommand` and `/includeCommand`

Reply:
agreed, I'll tidy that up and just hoist it all into the title declarations